simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/variable/attribute/AttributePath.java

```diff
@@ -0,0 +1,45 @@
+package com.evbox.everon.ocpp.simulator.station.component.variable.attribute;
+
+import com.evbox.everon.ocpp.v20.message.centralserver.Component;
+import com.evbox.everon.ocpp.v20.message.centralserver.GetVariableDatum;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetVariableDatum;
+import com.evbox.everon.ocpp.v20.message.centralserver.Variable;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Contains all data necessary to access arbitrary variable's attribute of the station
+ */
+@Value
+@Builder
+@AllArgsConstructor
+public class AttributePath {
```

Comment:
Should it be named AttributeProperties or something like that? Why AttributePath?

Reply:
Cause it represents a path to the attribute.
Similar to directory path, but in this case: `componentName\componentInstance\evseId\connectorId\variableName\variableInstance\attributeType`

Reply:
Could we put that in the class' javadoc?

Reply:
it's there, just in a different phrasing

Reply:
Okay... where can I find this in the source code?
componentName\componentInstance\evseId\connectorId\variableName\variableInstance\attributeType

Reply:
there's component, which contains component name, component instance and evse and connector id
variable which contains name and instance
attribute type